package.json

```diff
@@ -7,8 +7,8 @@
   "license": "MPL-2.0",
   "scripts": {
     "postinstall": "cd stumptown && npm install && git checkout package-lock.json",
-    "prettier-check": "prettier --check **/*.{js,jsx,scss,css,html}",
-    "prettier-format": "prettier --write **/*.{js,jsx,scss,css,html}",
+    "prettier-check": "prettier --check **/*.{js,jsx,json,scss,css,html}",
+    "prettier-format": "prettier --write **/*.{js,jsx,json,scss,css,html}",
```

Comment:
The original reason I included "css" and "jsx" is because they are included in this command

Reply:
Good catch! Would you mind removing .jsx and .css from that list?